qa/tasks/cephfs/test_readahead.py

```diff
@@ -1,25 +1,34 @@
 import logging
+import re
 from tasks.cephfs.fuse_mount import FuseMount
 from tasks.cephfs.cephfs_test_case import CephFSTestCase
 
 log = logging.getLogger(__name__)
 
 
 class TestReadahead(CephFSTestCase):
-    def test_flush(self):
-        if not isinstance(self.mount_a, FuseMount):
-            self.skipTest("FUSE needed for measuring op counts")
+    def get_op_read_count(self):
```

Comment:
Let's move this method to `mount.py`/`fuse_mount.py`/`kernel_mount.py`?

Reply:
Yeah, sounds good.

Reply:
Done. Thanks.